src/site/styles/_our-community.scss

```diff
@@ -0,0 +1,34 @@
+@import "vars";
+
+.our-community {
+
+  .row {
+    padding: 5vh 0;
+
+    &:first-child {
+      padding-top: 0;
+    }
+
+    &:not(:last-child) {
+      border-bottom: 1px solid $brand-dark;
+    }
+  }
+
+  .icon-container {
+    display: flex;
+
+    .icon {
+      width: 75%;
+      height: 100%;
+      fill: $brand-dark;
+    }
+  }
+
+  .page-hero {
+    background: url(/assets/about_hero_background.png);
```

Comment:
I am hopeless with assets. I just used the same hero image as About, but we can swap in whatever

Reply:
Yeah, we'll want to swap in the same image that's on the current community page